modules/apps/portal/portal-dao-test/src/testIntegration/java/com/liferay/portal/dao/db/test/DBTest.java

```diff
@@ -257,18 +263,47 @@ public void testAlterTableDropIndexedColumn() throws Exception {
 	@Test
 	public void testAlterTableName() throws Exception {
 		_db.runSQL(
-			StringBundler.concat("alter_table_name ", _TABLE_NAME, " DBTest2"));
+			StringBundler.concat(
+				"alter_table_name ", _TABLE_NAME_1, StringPool.SPACE,
+				_TABLE_NAME_2));
+
+		Assert.assertTrue(_dbInspector.hasTable(_TABLE_NAME_2));
 
-		Assert.assertTrue(_dbInspector.hasTable("DBTest2"));
+		_db.runSQL("DROP_TABLE_IF_EXISTS(" + _TABLE_NAME_2 + ")");
+
+		Assert.assertFalse(_dbInspector.hasTable(_TABLE_NAME_1));
+	}
+
+	@Test
+	public void testGetPrimaryKeyColumnNames() throws Exception {
+		_db.runSQL(
+			StringBundler.concat(
+				"create table ", _TABLE_NAME_2,
+				" (id1 LONG not null, id2 LONG not null, primary key (id1, ",
+				"id2))"));
 
-		_db.runSQL("DROP_TABLE_IF_EXISTS(DBTest2)");
+		Assert.assertArrayEquals(
+			new String[] {"id1", "id2"},
+			_db.getPrimaryKeyColumnNames(_connection, _TABLE_NAME_2));
+	}
 
-		Assert.assertFalse(_dbInspector.hasTable(_TABLE_NAME));
+	@Test
+	public void testGetPrimaryKeyColumnNamesIncorrectOrder() throws Exception {
+		_db.runSQL(
+			StringBundler.concat(
+				"create table ", _TABLE_NAME_2,
+				" (id1 LONG not null, id2 LONG not null, primary key (id1, ",
+				"id2))"));
```

Comment:
@achaparro I think in order to fully test this, we'll need to make sure that the expected primary keys are not in alphabetical order. Am I mis-reading this?

Reply:
@drewbrokke You are right, I missed that. Fixed here:
https://github.com/brianchandotcom/liferay-portal/pull/131487

Thanks.

Reply:
Great, thanks @achaparro!